Taskfile.yml

```diff
@@ -251,6 +252,32 @@ tasks:
         -target-backend=mongodb
         -no-xfail={{.NO_XFAIL}}
 
+  test-integration-yugabyte:
```

Comment:
At the moment all integration tests fail due to authentication with `2025-06-16T03:42:58.9427395Z ERROR	        	            	connection() error occurred during connection handshake: auth error: sasl conversation error: unable to authenticate using mechanism "SCRAM-SHA-256": (AuthenticationFailed) Authentication failed.`

See CI too https://github.com/FerretDB/FerretDB/actions/runs/15671263898/job/44142686066?pr=5258.

Looking a bit further, it seems to be failing with response from `documentdb_api_internal.scram_sha256_get_salt_and_iterations` returning `ok:0` with below payload.

```
{`ok`: 0, `iterations`: 0, `salt`: ``}
```

Reply:
Well, that's not good enough for that PR. We expect some/many tests to fail. But we don't expect all of them fail during setup.